### PR TITLE
Feature/use project options

### PR DESCRIPTION
**Pull request by ClausKlein**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# =============================================================================
+# PYTHON PACKAGES (PIP)
+# =============================================================================
+# USE (python3): sudo pip3 install -r <THIS_FILE_NAME>
```
Comment (papperlapapp) [inline requirements.txt]: Minor nitpick (due to use of ">="; and don't break your system python!):
```suggestion
# USE (python3): pip3 install -U -r <THIS_FILE_NAME>
```

File: standalone/source/main.cpp [hunk 2]
```diff
@@ -43,7 +43,7 @@ auto main(int argc, char** argv) -> int {
   auto langIt = languages.find(language);
   if (langIt == languages.end()) {
     std::cerr << "unknown language code: " << language << std::endl;
-    return 1;
+    return 0;
```
Comment (papperlapapp) [inline standalone/source/main.cpp:46]: Why no error?

File: GNUmakefile [hunk 3]
```diff
@@ -1,86 +1,66 @@
 # The "bootstrap" part clones repositories (with hardwired paths). With the
 # help of the below setting, we can redirect these clones to our internal
 # Gitlab mirror.
-#TODO export GIT_CONFIG_PARAMETERS='url.git@code.rsint.net:mirror/github.com/.insteadOf=https://github.com/'
+#TODO export GIT_CONFIG_PARAMETERS='url.git@code.example.com:mirror/github.com/.insteadOf=https://github.com/'
```
Comment (papperlapapp) [inline GNUmakefile]: Remove the whole comment til here.

File: README.md [hunk 4]
```diff
@@ -41,75 +43,88 @@ Alternatively you may use the [flexible project options](https://github.com/amin
...
 
-To cleanly separate the library and subproject code, the outer [CMakeLists.txt](CMakeLists.txt) only defines the library
-itself while the tests and other subprojects are self-contained in their own directories.  During development it is usually
-convenient to [build all subprojects at once](#build-everything-at-once).
+**To cleanly separate the library and subproject code, the outer [CMakeLists.txt](CMakeLists.txt) only defines the library
+itself while the tests and other subprojects are self-contained in their own directories.**
+During development it is usually convenient to [build all subprojects at once](#build-everything-at-once).
 
-### Build and test with cmake preset
+### Build the Release library, install it and test the exported CMake config Release package.
```
Comment (papperlapapp) [inline README.md]: Headings should be concise, so:
```suggestion
### Build, test and install the library
```

**Review by papperlapapp — COMMENTED** [review]
Workflows: looks like a lot of duplicated code there. Never worked with github workflows but it's certainly possible to factor out common parts and include it in others or maybe setting up matrix builds could elimininate code duplication.

**Comment by ClausKlein** [comment]
Yes,this will be a Future PR 🙏

File: .envrc [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+export CPM_USE_LOCAL_PACKAGES=YES
+export CPM_SOURCE_CACHE=${PWD}/.cache/CPM
+
+export LD_LIBRARY_PATH=${CMAKE_PREFIX_PATH}/lib
+export BUILD_DIR=${PWD}/build
+
+# The "bootstrap" part clones repositories (with hardwired paths). With the help of the below setting, you may redirect
+# these clones to an internal used Gitlab mirror.
+#TODO export GIT_CONFIG_PARAMETERS='url.git@code.example.com:mirror/github.com/.insteadOf=https://github.com/'
```
Comment (papperlapapp) [inline .envrc:18]: Remove these three comment lines.

**Review by papperlapapp — COMMENTED** [review]
IMHO too many changes in a single pull request.

**Pull request merged**